enhancements/update/phased-rollouts.md

```diff
@@ -0,0 +1,279 @@
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement request proposes phased update rollouts, allowing release administrators to declare a time window over which update recommendations are phased into per-cluster requests for stable channel [Cincinnati graphs][cincinnati].
+
+## Motivation
+
+We [currently][channel-semantics] has `fast-4.y` and `stable-4.y` channels for each 4.y series.
+That allows users who want to hear about releases and related update edges immediately after we declare them supported to do so via `fast-4.y`, while users who want to wait and see if we still recommend the edge after its seen more volume can do so via `stable-4.y`, as discussed [here][fast-vs-stable].
+However, clusters default to `stable-4.y` and users need to explicitly configure `fast-4.y` to populate that higher-risk cluster pool.
+This leaves the additional safety of `stable-4.y` ambiguous; when a cluster sees an available update recommended in `stable-4.y`, it's not clear now much traffic it has received.
+Conservative users are left to wait an arbitrary amount of additional time and guess about the number of other clusters which may be feeling out the new edge, or rely entirely on their own local testing.
+
+By smearing the update recommendation out over a time window, we address several issues:
+
+1. Reduced likelihood of a thundering herd of updating clusters swamping canonical image registries.
+    We currently lack built-in support for [automatic updates][automatic-updates], but some users have built analogous functionality as a higher-level driver for ClusterVersion.
+    Phased rollouts make it less likely to have many clusters simultaneously initiate a given update by removing the synchronized update recommendation.
+2. Reduced risk for stable promotion.
+    Instead of throwing a switch for all stable clusters simultaneously, release administrators would have the ability to augment meager `fast-4.y` data by trickling `stable-4.y` clusters over a new update edge.
+    Choices about which clusters in the stable pool get the recommended stable edge first can be made intelligently, and the riskier early slots can be rotated through the pool of stable clusters to share out the risk.
+    This means that by the time a given cluster receives an update recommendation, it's more likely that several other Red Hat customers have already been recommended and taken the given edge.
+
+### Goals
+
+* [The graph-data repository][graph-data] should grow a schema for declaring phased-rollout time windows.
+
+### Non-Goals
+
+* Deciding how to sort clusters within a given rollout window.
+    For an initial implementaiton, the "phased" rollout can be showing _all_ stable clusters the edges in question at the end of the configured windows, since that's effectively what we're doing today.
```

Comment:
Shouldn't the schema def be defining `stable clusters` even if only to support `_all_`?

Reply:
Did [this][1] reword address your concerns here?  The idea is that, for the initial policy-engine implementation, clients requesting stable channels would never see stable edges before the end of the window, and would always see stable edges after the end of the window, regardless of the cluster ID associated with the request. 

[1]: https://github.com/openshift/enhancements/pull/427#discussion_r492927259

Reply:
I understood the semantics of _all_. What I didn't fully understand, but do now after you explained it in our meeting, was how a given cluster would get the indication to upgrade. And my comment above was that I thought a brief explanation of that mechanism -the policy engine, etc. - would be helpful in understanding the details of the schema changes and the alternatives.